packages/components/src/components/password-input/password-input.jsx

```diff
@@ -4,47 +4,38 @@ import PropTypes from 'prop-types';
 import Icon from '../icon';
 import Input from '../input';
 
-class PasswordInput extends React.Component {
-    state = {
-        show_password: false,
-    };
+const PasswordInput = ({
+    className, // Must not be passed to Input as the only trailing icon should be the visibility icon
+    ...otherProps
+}) => {
+    const [show_password, setShowPassword] = React.useState(false);
```

Comment:
```suggestion
    const [is_password, setShowPassword] = React.useState(false);
```

Reply:
@reza-fs Since this state is using for showing or hiding the password to the user, I believe `is_password` is not an appropriate name, however, because it is boolean `show_password` is not either. I suggest `should_show_password`, what do you think?